### PR TITLE
Wrap relu, leaky_relu, tanh and sigmoid

### PR DESCRIPTION
**Pull request by shendiaomo**

### CODE REVIEW

File: tensor.go [hunk 1]
```diff
@@ -116,6 +148,26 @@ func MM(a, b Tensor) Tensor {
 	return Tensor{&t}
 }
 
+// LeakyRelu returns leaky relu of the tensor according to negativeSlope
+func LeakyRelu(t Tensor, negativeSlope float64) Tensor {
+	return t.LeakyRelu(negativeSlope)
+}
+
+// Relu returns relu of the tensor
+func Relu(t Tensor) Tensor {
```
Comment (QiJune) [inline tensor.go:157]: It seems that we provide two kinds of API:

First way:

```
a = RandN((100, 10)
b = a.Relu()
```

Second way:

```
a = RandN((100, 10)
b = Relu(a)
```

Which API do we prefer to use?

Comment (shendiaomo) [reply tensor.go:157]: The C++ and Python frontend also support these two styles of APIs. We'd better support both.

Comment (Yancey1989) [reply tensor.go:157]: Agree with the second way, it's much more like Python API
``` python
def forward(self, x):
    x = self.conv1(x)
    x = F.relu(x)
    ....
```

And some other function API maybe have more then one Tensor, e.g. 
``` python
torch.stack(tensors, dim=0, out=None) → Tensor
```
https://pytorch.org/docs/stable/generated/torch.stack.html#torch.stack

Comment (QiJune) [reply tensor.go:157]: It seems that `numpy` also support two kinds of APIs:

```python
import numpy as np
a = np.array([1.0, 2.0])
print(a.sum())
print(np.sum(a))
```

So, I feed well that we provide both.

File: functions_test.go [hunk 2]
```diff
@@ -33,3 +33,16 @@ func ExampleMM() {
 	_ = z
 	// Output:
 }
+
+func ExampleRelu() {
```
Comment (wangkuiyi) [inline functions_test.go:37]: This should be `TestRelu(t *testing.T)` instead of `ExampleRelu()` because we need to check the returned values, right?
